udev/49-vtlinkii.rules

```diff
@@ -0,0 +1,3 @@
+# 30cc:9527 Essemi ESLinkII
+ATTR{idVendor}=="30cc", ATTR{idProduct}=="9527", MODE="666"
```

Comment:
`SUBSYSTEM` restricts it so only devices of the USB subsystem that also have the VID and PID will be allowed. The reality is this distinction probably doesn't need to be made. I'd be happy to include the subsystem but the subsystem this particular debugger belongs to is tty. Also, the `:=` is basically a non-blocking assignment so it will only change the mode if it's not overwriting something. I had issues with this so I basically did a forcing equals

Reply:
Thanks for the explanation. Feel free to keep as it is since it works for you.